### DISCUSSION TITLE
Onion Site Not Found / Sito Onion non trovato

### DISCUSSION THREAD

**Discussion by gabrielelakhal** [globaleaks/globaleaks-whistleblowing-software#3814] [Q&A]
Good evening,

I have made several updates to Ubuntu, and after restarting, the various whistleblowing sites are all operational and accessible through the URLs set on DNS. However, when I try to access the different panels through onion URLs or the general dashboard where I manage the different instances, the Tor browser gives me the following error:

ENGLISH (translated)
"Onion Site Not Found
Details: 0xF0 — The requested Onion service descriptor cannot't be found in the hashring, so the service is unreachable by the client."

ITALIAN (original error message)
"Sito Onion non trovato
Dettagli: 0xF0 — Il descrittore del servizio Onion richiesto non può essere trovato nell'hashring, perciò il servizio non è raggiungibile dal client."

Has this happened to any of you?
How did you resolve it?
Can you help me?

Thank you,

Gabriele

**Comment by davidebiani** [comment]
I have encountered the same problem after updating to version 4.13.18. I am currently in the process of attempting to understand the issue, but I find myself facing some challenges in doing so.

@evilaliv3, I would appreciate your insights and thoughts on this matter.

**Comment by brassy-endomorph** [comment]
Bumping this in that we have this issue too. @evilaliv3 could you make respond here since it seems like several people have sites that are down and your Troubleshooting help docs are basically useless. Also globaleaks doesn't output any logs other than "starting" and "stopping" so we can't even handle this on our own.

**Reply by evilaliv3** [reply]
Thank you @brassy-endomorph; you seems to have a great good energy and probably nice ideas for improvement would you like to work on this; whould you like to contribute to the software directly?

**Comment by brassy-endomorph** [comment]
Cross posted this as a bug in hope someone sees it https://github.com/globaleaks/GlobaLeaks/issues/3847

**Comment by evilaliv3** [comment]
Thank you all except @brassy-endomorph for your patience :)

I've still not answered cause unfortunately i've not been able to reproduce the issue that seems to happen time to time quite randomply.
